scripts/deploy_benchmark.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+RUNNER_URL=$1
+current_dir=$(dirname "${BASH_SOURCE[0]}")
+parent_dir=$(dirname "${current_dir}")
+path_to_parent="$( cd "${parent_dir}" && pwd )"
+temp_dir="${path_to_parent}"/temp
+
+# Delete temp dir if it exists
+rm -rf "${temp_dir}"
+
+# Clone Launcher Repo
+git clone --branch eq-3425-terraform-benchmark --depth 1 https://github.com/ONSdigital/eq-survey-runner-benchmark.git "${temp_dir}"/eq-survey-runner-benchmark
+
+cd ${temp_dir}/eq-survey-runner-benchmark
+
+helm tiller run \
+    helm upgrade --install \
+    runner-benchmark \
+    k8s/helm \
+    --set host=${RUNNER_URL} \
+    --set image.repository=${DOCKER_REGISTRY}/eq-survey-runner-benchmark \
+    --set image.tag=${IMAGE_TAG}
+
+# Delete repo
+rm -rf "${temp_dir}"
```

Comment:
I can see that the temp directory stuff is copied from other repos, but we can simplify this script by using the `mktemp` command to create the temp directory. For reference, we use it in https://github.com/ONSdigital/eq-survey-runner/blob/v3/scripts/load_templates.sh.